src/test/java/com/sendgrid/TestRequiredFilesExist.java

```diff
@@ -13,18 +13,6 @@
         assertTrue(dockerExists);
     }
 
-    // ./docker-compose.yml or ./docker/docker-compose.yml
```

Comment:
Can you please just comment out these tests for now? Eventually we want them to pass. Thanks!

Reply:
No problem, changes are commited :+1:

Reply:
Actually, should i only re-add the docker-compose ones or USE-CASES.md and .env-sample as well?